main/ferm/APKBUILD

```diff
@@ -1,15 +1,15 @@
 # Contributor: Michael Mason <ms13sp@gmail.com>
 # Maintainer: Michael Mason <ms13sp@gmail.com>
 pkgname=ferm
-pkgver=2.4
+pkgver=2.4.1
 pkgrel=0
 pkgdesc="firewall configuration tool"
 url="http://ferm.foo-projects.org/"
 arch="noarch"
 license="GPL"
 depends="perl iptables"
 options="!check"
-source="http://ferm.foo-projects.org/download/$pkgver/$pkgname-$pkgver.tar.gz"
+source="http://ferm.foo-projects.org/download/2.4/$pkgname-$pkgver.tar.xz"
```

Comment:
Merged, but please try to avoid hardcoding version information in `$source` in the future. I replaced `2.4` with `${pkgver%%.*}`.